src/zenml/orchestrators/utils.py

```diff
@@ -130,17 +130,18 @@ def execute_step(
         execution_info = tfx_launcher.launch()
         if execution_info and get_cache_status(execution_info):
             if execution_info.exec_properties:
-                pipeline_run_id = execution_info.pipeline_run_id
+                execution_info.pipeline_run_id
```

Comment:
If I am not mistaken, previously, this was used to extract the `pipeline_run_id`. As this is not the case anymore, can we remove this line?

Reply:
sure